Modules/Core/Transform/test/itkComposeScaleSkewVersor3DTransformTest.cxx

```diff
@@ -0,0 +1,554 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+/**
+ *
+ *  This program illustrates the use of Similarity3DTransform
```

Comment:
Similarity3DTransform->ComposeScaleSkewVersor3DTransform

Reply:
The same change should probably be applied to the original class too?

Reply:
Changed in new and in old/original class too.